pkg/yurthub/storage/disk/storage.go

```diff
@@ -87,6 +88,21 @@ func (ds *diskStorage) Create(key string, contents []byte) error {
 			}
 			return err
 		} else if info.IsDir() {
+			// ensure that there are no files left locally before create an dir
+			if fileInfoList, err := ioutil.ReadDir(keyPath); err != nil {
```

Comment:
`ioutil.ReadDir(keyPath)` will return a sorted list files, but we only need to list all file names, so how about use the following code:
```
    f, err := os.Open(keyPath)
    if err != nil {
        return err
    }
    defer f.Close()
    names, err := f.Readdirnames(-1)
    if err != nil {
        return err
    }
```

Reply:
The code is updated, the Create(..) no longer cleans up old files, and the related functions are placed in the Replace(..).